server/src/main/java/org/apache/iotdb/db/mpp/plan/statement/sys/sync/ShowPipeStatement.java

```diff
@@ -28,14 +28,24 @@ public class ShowPipeStatement extends ShowStatement implements IConfigStatement
 
```

Comment:
```suggestion
  public ShowPipeStatement() {
    super();
    statementType = StatementType.SHOW_PIPES;
  }
```

This will be used in the auth module.